### DISCUSSION TITLE
[draft-v1] Figure 4: Committers per month

### DISCUSSION THREAD

**Discussion by pllim** [astropy/astropy-v5.0-paper#51] [Figure comments] [answered]
The plot is based on number of individual commits, not merge commits. Hence, it is curious that even individual _committers_ (the people) is so motivated by feature freezes: Who are these people and why?

Furthermore, it is a little concerning that the spikes of _committers_ get higher with time. This tells me that people are getting bolder and bolder to wait till last minute to push commits. For instance, number of _committers_ jumped from 20 to 40 (100% increase!) in the feature freeze that happened in early 2021. I know the usual suspects but I do not know there are 20 of them.

**Comment by hamogu** [comment]
> For instance, number of committers jumped from 20 to 40 (100% increase!) in the feature freeze that happened in early 2021. 
That may in fact warrant it's own mention in the section on "challenges" since in an increasingly complex system (more downstream packages over time) pushed stuff in just before the deadline is making it harder for those doing the releases and the testing.

**Reply by adrn** [reply]
Feel free to open a pull request with some added text!

**Comment by bmerry** [comment]
What timestamp is used - the original timestamp of the commit, or the time at which the commit was merged?

**Reply by adrn** [reply]
The timestamp of the commit was used here

**Comment by adrn** [comment, marked as answer]
Just to check @pllim: Do you want to take any action with this discussion? I agree this is interesting, but would you like to add any text to the paper or did you want to discuss on here?

**Reply by pllim** [reply]
No, I think the paper is too long as it is. Not sure if this is worth discussing in *this* paper.
